src/OMSimulatorLib/Flags.h

```diff
@@ -106,6 +107,7 @@ namespace oms3
 
     const std::vector<Flag> flags = {
       {"", "", "FMU or SSP file", re_filename, Flags::Filename, false},
+      {"--clearAllOptions", "", "Reset all flags to efault vlaues", re_void, Flags::ClearAllOptions, false},
```

Comment:
Type `efault vlaues`.
Should be `re_bool` instead of `re_void`.

Reply:
Thanks, it was actually a typo in the description.

Reply:
There is another typo in the description of the flag.